profiles/models.py

```diff
@@ -7,6 +7,24 @@
 from jsonfield import JSONField
 
 
+class Employment(models.Model):
+    """
+    A user work_history
+    """
+    city = models.TextField()
+    company_name = models.TextField()
+    country = models.TextField()
+    industry = models.TextField()
+    position = models.TextField()
+    state_or_territory = models.TextField(blank=True, null=True)
+    end_date = models.DateTimeField()
```

Comment:
Can you change this and `start_date` to `models.DateField` instead, since there is no time component we care about?